languages/rust/install.sh

```
@@ -6,7 +6,7 @@ source "$(dirname "$0")/../../bin/common.sh"
 
 install_rust() {
   log_info "Installing Rust using rustup..."
-  curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+  run_remote_installer https://sh.rustup.rs sh "{}" -y
   source "$HOME/.cargo/env"
```

Shellcheck (0.11.0)

[info] Not following: ./.cargo/env was not specified as input (see shellcheck -x).

(SC1091)